README.md

```diff
@@ -1,170 +1,508 @@
+(_instructions given below_).
+
+<!--
+> Point People to the "Landing Page" (once it's ready):
+https://github.com/dwyl/github-backup/issues/55
+-->
+
+
+## Set Up _Checklist_ on `localhost`
+
+This will take approximately **15 minutes** to complete. <br />
+(_provided you already have a GitHub and AWS account_)
+
+### Install Dependencies
+
+To run the App on your `localhost`,
+you will need to have **4 dependencies** _installed_:
+
++ **Elixir**:
+https://github.com/dwyl/learn-elixir#installation
++ **PostgreSQL**:
+https://github.com/dwyl/learn-postgresql#installation
++ **Phoenix**:
+https://hexdocs.pm/phoenix/installation.html
++ **ngrok**: https://github.com/dwyl/learn-ngrok#1-download--install <br />
+(_so you can share the app on your `localhost` with GitHub via `public` URL_)
+
+Once all the necessary dependencies are installed,
+we can move on.
+(_if you get stuck,
+  [please let us know](https://github.com/dwyl/github-backup/issues)_!)
+
+### Clone the App from GitHub
+
+Run the following the command to `clone` the app from GitHub:
+```sh
+git clone https://github.com/dwyl/github-backup.git && cd github-backup
+```
+
+### Required Environment Variables
+
+To run the project on your `localhost`,
+you will need to have the following Environment Variables defined.
+
+> _**Note**: if you are `new` to "Environment Variables", <br />
+**please read**_:
+[github.com/dwyl/**learn-environment-variables**](https://github.com/dwyl/learn-environment-variables)
+
++ `APP_HOST` - the hostname for your app e.g: `localhost` or `gitbu.io`.
++ `PRIVATE_KEY` - The RSA private key for your GitHub App.
+_See below for how to set this up_.
++ `GITHUB_APP_ID` - The unique `id` of your GitHub App. _See below_.
++ `GITHUB_APP_NAME` - the name of your GitHub App. _See below_ (_Step 1_).
++ `SECRET_KEY_BASE` - a 64bit string used by Phoenix for security
+(_to sign cookies and CSRF tokens_). See below for how to _generate_ yours.
++ `AWS_ACCESS_KEY_ID` - your AWS access key (_get this from your AWS settings_)
++ `AWS_SECRET_ACCESS_KEY` - your AWS secrete access key
++ `S3_BUCKET_NAME` - name of the AWS S3 "bucket" where issue comments
+will be stored.
+
+
+#### Copy The `.env_sample` File
+
+The _easy_ way manage your Environment Variables _locally_
+is to have a `.env` file in the _root_ of the project.
+
+_Copy_ the _sample_ one and update the variables:
+
+```sh
+cp .env_sample .env
+```
+
+Now update the _values_ to the _real_ ones for your App. <br />
+You will need to _register_ a GitHub app,
+so that is what we will do _below_!
+
+> _Don't worry, your sensitive variables are **safe**
+as the `.env` file is "ignored" in the_
+[`.gitignore` file](https://github.com/dwyl/github-backup/blob/778d1e311a37564721989e842412880994eb2b5d/.gitignore#L28)
+
+
+
+### Create a GitHub Application
+
+The role of the Github application is to
+send **notifications** when **events** occur in your repositories. <br />
+For example you can get be notified
```

Comment:
Typo, either needs to be 'For example you can ~get~ be notified' or 'For example you can get ~be~ notified'.